packages/select/src/useSelect.ts

```diff
@@ -521,15 +521,21 @@ export const useSelect = (props, states: States, ctx) => {
   }
 
   const scrollToOption = option => {
-    const target = Array.isArray(option) ? option[0]?.$el : option.$el
+    const getEl = option => {
```

Comment:
Ssems to me that this function will only be called once. The only thing different is that you need to pick options out of multiple options, I don't see the benefit of encapsulating a function for that

Reply:
Yes, I will extract it to the outside.